spec/requests/todos_spec.rb

```diff
@@ -0,0 +1,63 @@
+require "rails_helper"
+
+RSpec.describe "API V1 ToDo", type: :request do
+  describe "GET /api/v1/todos" do
+    it "Retrieves a list of Todo items" do
+      headers = { "ACCEPT" => "application/json" }
+      get "/api/v1/todos", headers: headers
+      expect(response.status).to eq(200)
```

Comment:
There are (I believe) some nice status matchers in Rspec you can use to make these statements a bit easier to read:

```ruby
expect(response).to have_http_status(:ok)
```

I find this to be a bit more expressive and clear.